resources/js/src/app/components/itemList/ItemSearch.js

```diff
@@ -50,7 +50,8 @@ Vue.component("item-search", {
 
             this.$store.commit("setItemListSearchString", urlParams.query);
 
-            this.$refs.searchInput.value = urlParams.query ? urlParams.query : '';
+            let rawQuery = urlParams.query ? urlParams.query : '';
```

Comment:
In `resources/js/dist/filters-component.js` you used `var` statement, we should be consistent.

Reply:
That's because `filters-component.js` is the compiled one and should work for all browsers. I didn't wrote that, `gulp` compiled it for me.